src/c4group/CStdFile.h

```diff
@@ -20,7 +20,7 @@
 #ifndef INC_CSTDFILE
 #define INC_CSTDFILE
 
-#include <stdio.h>
+#include <cstdio>
```

Comment:
I agree, but this is something `modernize-deprecated-headers` did.
I guess a working build afterwards should be enough to ensure that nothing went wrong. Thoughts?